azure-ts-appservice/sas.ts

```diff
@@ -0,0 +1,37 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import * as azurestorage from "azure-storage";
+
+// Given an Azure blob, create a SAS URL that can read it.
+export function signedBlobReadUrl(
```

Comment:
We'll likely want to move this code out into `@pulumi/azure` itself as a helper since it appears to be needed broadly to support App Service and Functions.  I've opened https://github.com/pulumi/pulumi-azure/issues/70 to track this, which we can tackle separately.